classes/Path.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace OpenCFP;
+
+class Path
+{
+    private $path;
+    /**
+     * @var Environment
+     */
+    private $env;
+
+    public function __construct($path, Environment $env)
+    {
+        $this->path = $path;
+        $this->env  = $env;
+    }
+
+    public function basePath()
+    {
+        return $this->path;
+    }
+
+    /**
+     * Get the configuration path.
+     *
+     * @return string
+     */
+    public function configPath(): string
+    {
+        return $this->basePath() . "/config/{$this->env}.yml";
+    }
+
+    /**
+     * Get the uploads path.
+     *
+     * The production path is separated to make it easier for prod environments where it is
+     * not possible to store files on the disk.
+     *
+     * @return string
+     */
+    public function uploadPath(): string
+    {
+        if ($this->env->isProduction()) {
+            return $this->basePath() . '/web/uploads';
```

Comment:
The way it looks right now, there's no difference between the results returned from this method, how about only adding the condition when we actually return a different value?